app/models/vm_or_template/operations.rb

```diff
@@ -118,6 +118,16 @@ def validate_vm_control
       msg ||= _("Provider doesn't support vm_destroy") unless ext_management_system.respond_to?(:vm_destroy)
       unsupported_reason_add(:terminate, msg) if msg
     end
+
+    supports :suspend do
+      msg = unsupported_reason(:control) unless supports_control?
+      msg ||= _("The VM is not powered on") unless vm_powered_on?
+      unsupported_reason_add(:suspend, msg) if msg
+    end
+  end
+
+  def vm_powered_on?
+    current_state == "on"
```

Comment:
please move this to `vm/operations/power.rb`
also add 
```
_("Provider doesn't support vm_suspend") unless ext_management_system.respond_to?(:vm_suspend)
```
like in `supports :terminate` above.

The reason is that templates can be terminated / destroyed, but not suspended. Hence we move this to vm.